pkgs/development/python-modules/spectra/default.nix

```diff
@@ -0,0 +1,25 @@
+{
+  buildPythonPackage,
+  colormath,
+  fetchPypi,
+  lib,
+}:
+
+buildPythonPackage rec {
```

Comment:
- I think the boolean attribute `pyproject` should be added.
- Tests? What about them?
- While there are no enforced rules about the attribute order, make them consistent within all the files you're introducing. (e.g., `pname` then `version`).